Data/Graph.hs

```diff
@@ -270,8 +270,8 @@ type Vertex  = Int
 type Table a = Array Vertex a
```

Comment:
There should probably be a Haddock line explaining that this is included for backwards compatibility.

Reply:
Done.